<!-- Launch file to detect charuco markers -->

<launch>
    <group ns="saf">
    <arg name="marker_size_ratio" default="0.05"/>
    <arg name="image_markers" default="vision/image_markers"/>
    <arg name="markers" default="vision/markers"/>

    <node name="marker_displacement" pkg="irob_vision_support"
            type="marker_displacement_one.py" output="screen" >
            <param name="marker_size_ratio"
                            type="double" value="$(arg marker_size_ratio)" />
                <remap from="qrkam" to="vision/target" />



    </node>
    </group>
</launch>

